package.json

```diff
@@ -6,8 +6,10 @@
 	"bugs": "https://github.com/forcedotcom/sfdx-scanner/issues",
 	"dependencies": {
 		"@oclif/core": "^3.3.2",
-		"@salesforce/code-analyzer-core": "^0.8.0",
-		"@salesforce/code-analyzer-engine-api": "^0.6.0",
+		"@salesforce/code-analyzer-core": "^0.9.0",
+		"@salesforce/code-analyzer-engine-api": "^0.7.0",
```

Comment:
With the sample engine plugin removed... are we at a point where we can remove this dependency. If it is still needed as a test dependency, that's fine. But ideally the cli shouldn't depend on the engine-api for its main source dependencies.

Reply:
[`EnginePluginsFactory.ts`](https://github.com/forcedotcom/sfdx-scanner/pull/1546/files#diff-0aa7c410e4a9da7abce0527daf9e7f9e1eaa7a85d2b955311b8f78bbd467dbb5R1) still needs to import `EnginePlugin` for its `.create(): EnginePlugin[]` declaration. Is there an alternate place it can import that from?

Reply:
Ah I see... I mentioned this in the engine design doc... that yeah if you want to store an array first... you will indeed need to bring in the interface. Alternatively, we would just construct the plugins inline when we added them to the core module.

But I suppose it doesn't hurt to have the engine api as a dependency... it will just be annoying to have to update its version if/when we update the engine-api on the CLI side.